### ISSUE TITLE
Support for description node when adding new shares

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#54]
```
Please implement support for the description node in 
LinkedInApiClient.postShare() methods.

See http://developer.linkedin.com/docs/DOC-1212 for more info.

Thanks!


```

Original issue reported on code.google.com by `ectjab...@gmail.com` on 17 May 2011 at 7:01

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting it. I will update it in the next build.
```

Original comment by `nabeelmukhtar` on 17 May 2011 at 7:30
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Cool. Thanks.
```

Original comment by `ectjab...@gmail.com` on 17 May 2011 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
This defect is fixed in build 1.0.389
```

Original comment by `nabeelmukhtar` on 10 Jun 2011 at 3:30
- Changed state: **Fixed**
